repository-data/application/src/main/resources/hcm-content/content/documents/administration/labels/pages/itinerary.yaml

```
@@ -260,59 +260,61 @@
+          range of time like: 9:00am to 13:00pm', '', '', '', '', '', '', '', '',
+        '']
       resourcebundle:id: itinerary
       resourcebundle:keys: [stop.nearby-stay, stop.nearby-eat, stop.title, start-finish,
         days, distance, themes, miles-abbreviation, kilometres-abbreviation, transport,
         day, miles, kilometres, highlights, areas-covered, stop.no-stop, stop.time-to-explore,
         theme, stop.opening, stop.hour, stop.hours, stop.usually, stop.provisionally,
         stop.today, stop.to, stop.open, stop.closed, stop.close.soon, stop.temporarily-closed,
-        stop.and, end-itinerary, map-view, list-view]
+        stop.and, end-itinerary, map-view, list-view, keyfacilities.title]
       resourcebundle:messages: [Nearby places to stay, Nearby places to eat, Stop,
         Start / Finish, Days, Distance, Main themes, mi, km, Transport, Day, Miles,
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 32 but found 8

(indentation)